clockwork_web/static/locales/fr/LC_MESSAGES/messages.po

```diff
@@ -18,56 +18,248 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.10.3\n"
 
-#: clockwork_web/user.py:143
+#: clockwork_web/user.py:168
 msgid "could not modify api key"
 msgstr "la clef API n'a pas pu être modifiée"
 
-#: clockwork_web/user.py:156
+#: clockwork_web/user.py:181
 msgid "could not modify update key"
 msgstr "la clef de mise à jour n'a pas pu être modifiée"
 
-#: clockwork_web/browser_routes/jobs.py:137
-msgid "Field 'relative_time' cannot be cast as a valid integer: %(time1)."
-msgstr ""
-"Le champ 'relative_time' n'a pas pu être converti en nombre entier "
-"valide: %(time1)"
+#: clockwork_web/browser_routes/admin.py:49
+msgid "Authorization error."
+msgstr "Problème d'autorisation"
+
+#: clockwork_web/browser_routes/clusters.py:62
+msgid "This cluster is not known."
+msgstr "La grappe de calcul (/le cluster) demandée n'a pas été trouvée."
```

Comment:
I would remove "(/le cluster)" because it's fine. If we were explicit about the argument's name in English, we could mention it in French, but here in English we're just talking about "this cluster".

Maybe that's a justification for changing the English message as well, but that's not really the objective with this PR for now.